functional_data/lib/src/annotations.dart

```diff
@@ -1,7 +1,37 @@
 import 'package:collection/collection.dart';
 
 class FunctionalData {
-  const FunctionalData();
+  const FunctionalData({
+    this.generateCopy,
+    this.generateCopyWith,
+    this.generateCopyUsing,
+    this.generateLenses,
+  });
+
+  /// Whether the generator should generate the copy methods.
+  ///
+  /// When set, it overrides the settings of [generateCopyWith] and [generateCopyUsing].
+  final bool? generateCopy;
+
+  /// Whether the generator should generate the copyWith method.
+  ///
+  /// Defaults to `true`.
+  final bool? generateCopyWith;
+
+  /// Whether the generator should generate the copyUsing method.
+  ///
+  /// When set to `true`, a helper class will also be generated (called `ClassName$Change`).
```

Comment:
It is not about the helper class, it is about the `copyUsing` method.

Reply:
Ow yes. I meant "a (helper) class that implements copyUsing functionality". I will rephrase it